lib/status/status.go

```diff
@@ -129,6 +134,14 @@ func FromCluster(ctx context.Context, operator ops.Operator, cluster ops.Site, o
 	}
 
 	status.State = cluster.State
+
+	// Collect information from alertmanager
+	if dnsAddr != "" {
```

Comment:
Is this a good way to determine whether we need to collect alertmanager alerts or not? Why does it depend on passing DNS address? Maybe explicit flag would be better?

Also, I wonder how does it work on older clusters? I.e. if you run `gravity status` using this binary on an older cluster - what would happen?

Reply:
Is this a good way to determine whether we need to collect alertmanager alerts or not? 
- It's not about determining whether or not to collect the alertmanager, it's to prevent undefined behaviour if the status is called without the DNS configuration.

Why does it depend on passing DNS address?
- Alertmanager runs as a kubernetes serice/pods. So we need to discover the service address in use, which requires connecting to the dns service discovery. Alternatively it might be possibly to set alertmanager to a static IP, but we would still need dynamic config to calculate the IP based on the service range used, and I generally prefer to let kubernetes be the authority on IP assignments.

Maybe explicit flag would be better?
- Again, my thinking was it's not about turning on/off, more that if the config is incomplete that we don't run into undefined behaviour. 

Also, I wonder how does it work on older clusters?
It should fail to resolve the service discovery address, and I believe throw an error similar to if etcd is down or it's unable to reach planet. Although I'm not sure under what circumstances a gravity 6 binary will have someone run `gravity status` against a non 6+ cluster??

Reply:
I can easily imagine someone running gravity status using new binary on an old cluster. Say, when upgrading from pre-6.0 to 6.0+ and using the new binary. I'm sure you can imagine a few too, customers are unpredictable as you know :)

Anyway, it makes sense about the dns addr, but we need to make sure that the failure is handled gracefully here and not fails the whole gravity status output.